src/EFCore.Relational/Extensions/RelationalQueryableExtensions.cs

```diff
@@ -158,12 +159,49 @@ public static DbCommand CreateDbCommand([NotNull] this IQueryable source)
                 Expression.Constant(arguments));
         }
 
+        /// <summary>
+        ///     <para>
+        ///         Returns a new query in which the collections in the query results will be loaded in single database query.
```

Comment:
Talking about "returning a new query" may be a bit confusing to users... How about something like:

```suggestion
        ///         Configures the query so that collections in the query results will be loaded in single database query.
```

(same remark for other queryable extensions, e.g. AsTracking)

Reply:
We use terminology returns because it returns a new Expression tree.

Reply:
I think what matters here in the docs is what users need to know... From a user perspective there aren't two queries here, just one being marked/configured for query splitting...

Reply:
To be consistent with rest of Queryable terminology,
How about this?
```suggestion
        ///         Returns a new query which is configured to load the collections in the query results in single database query.
```

Reply:
I'm fine with it either way - including with the original, I just think we should document based on what users need to know, that's all. But OK to merge as you want.